modules/shortcodes/js/gist.js

```diff
@@ -0,0 +1,26 @@
+;(function($, undefined) {
+    gistStylesheetLoaded = false;
+
+    var gistEmbed = function( data ) {
+        $('.gist-oembed').each(function(i, el) {
+            var url = 'https://gist.github.com/' + $(el).data('gist');
+            $.ajax({
+                url: url,
+                dataType: 'jsonp'
+            }).done(function( response ) {
+                $(el).replaceWith( response.div );
+
+                if ( ! gistStylesheetLoaded ) {
+                    $('head').append(
+                        "<link rel='stylesheet' href='" + response.stylesheet + "' type='text/css' />"
+                    );
+
+                    gistStylesheetLoaded = true;
+                }
+            });
+        });
+    };
+
+    $(document).ready(gistEmbed);
+    $('body').on('post-load', gistEmbed);
+})(jQuery);
```

Comment:
Looks nice, but needs a little work to make it compliant with the WP JS standards: https://make.wordpress.org/core/handbook/best-practices/coding-standards/javascript/
Can you please go one step further? Thanks :)